### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by JuanBohorquezPerficient**

### CODE REVIEW

File: cypress/page/download.page.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
...fzfR0NNEuMr0i3hb6/T8yvhE/9lx1EP516hcafKcf35hSFc1+ie4SWyRlzaSbd8QWfPTriT/tX3e4jJOXbL1f5pCbRrjjNbs0RxS6hQi+yvQx/EtbNu14Fy1ClzlJeFV9BWivXJm9MOMzLixTq4y4k+kkOx5t6lFx7/wAPqWmYSj+NjVIkvAPhK4SjO0BZA+Aggsye5hCO6W3VpW2c2eROXyxpdgep1Ll4dnYIRk3Z6OKDW5PYyMaAlzDjO1RTQi1+wZA2WuYaSfYIoWFxEcStt78gAkYt8hscTr98hcMjXIKOZ9QE7CjLaily3KyVzVotZlW6AVF8NFyihan38wgCiJGz2fKO8Xs+jTavu7DJZO8aJlHkqOpOD6OflGKfqZ1iy3081G/Mfo9VxJLql6mkukzjc5QdNGfFpefE7vp08Nx0FGOy27ugUlsYMkMi/Gq7duXePoSufbNaTbdu13Kmv1F5IpS/8lPsdPfwMkc87pTt9nTyYz4O3du+9OxXZfDi9sblTVLiTvq4/ejPknXVPw2Q5YsnSQvJhydil5R+gmVGvyiY5x2bf/a16MbGre8afck/VGKHOq8rr7h6i10i+9WvIVluG6Hww47q732+bc0xtcvmXe1Zy45K5wj9f2g56p38jY00KWKT/wAm3LOMueO//ltCp6bHfKUb8KNGJNreUW/7Rc8nNOVd9tL6oZlFtaRqxSiklxRfml9Bvgr9DEpQa3kn5K/p+hMS/kUq7nHh+5XIyeM1vJ/TL0v7FQyJ9GvFNEWSXVet/lYy33eo0ZtV/wBFScr2jfmQZb7PsQAv9HmuZRRDE9goNPYhBAVKikQgwCUXe/3Df8SaVLvdkIIQtLmFCiiDGNUu7oA4r92QgElSxdbLtpbkIJjRXEWns1+0yEAdBaXJwyT/AHXU7yiQhpA4vLXTL4RE1JbKEa8ef0IQpnLGVMyT0bVt1XOq28LW4yGqpbwddfmTLIQ9PR0xfsX3FRzRlb95Lfu/wUsi4qTTffFqyECzTglY3PwWlJfcHEsfJLv6/voQgzNR+3sd7u18rvrvfLxF5Fwq+Bd/L8yEKoxUnyoPHNtcWy7qELJFv5oxve+f2LISzWKVsv4Xs4fJNP1Bek4VtOS89iyA0R7JXRax5NuHLfiv8BvPOC+aKfen+RCDrVhF8pcWiL2lHsf0IQhHNnV9LjP/2Q==";
```
Comment (holgiosalos) [inline cypress/page/download.page.ts]: is this needed?

File: cypress/page/download.page.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    public visitPage(){
+        cy.visit(this.testUrl);
+    }
+
+    public downloadFile(){
+        cy.get(this.downloadBtn).click({force:true});
+        cy.wait(1000);
```
Comment (holgiosalos) [inline cypress/page/download.page.ts]: are you sure that the wait is needed?

Comment (JuanBohorquezPerficient) [reply cypress/page/download.page.ts]: Oh sorry, I forgot to delete those lines
 I'm in my way to erase that stuff
thx, Holmes

File: cypress/e2e/iframe.cy.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import {IframePage} from "../page/index";
```
Comment (holgiosalos) [inline cypress/e2e/iframe.cy.ts]: as this is for the next step, could you deleted and wait until merge the changes related to upload-download page

Comment (JuanBohorquezPerficient) [reply cypress/e2e/iframe.cy.ts]: Ok, I've deleted all the stuff from the next step, thx

**Review by holgiosalos — APPROVED** [review]
For next PR you can change the branch, it does not have to be always the setup branch. It is only a suggestion. You can continue using it, but it is kindof weird see a lot of commits in each PR

**Pull request merged**